challenges/07_bootcamp_lists/box_office_challenge.py

```diff
@@ -8,31 +8,71 @@
 # Note: Print your new variable to see the list of movies
-print()
+# assigned variable remain-50_list and modified movies_str by splitting
+remaining_50_list= movies_str.split(',')
+print(remaining_50_list)
 
-# TODO: Hmm. Looks like "remaining_50_list" contains the movie "The Matrix" which is also in the "top_50_list". Let's remove this duplicate title from "remaining_50_list".
-# Note: Make sure to double check the list you are removing from!
 print()
 
 
-# TODO: We're down to 49 movies in "remaining_50_list". Let's add a movie to the list called "Anywhere But Here"
-# Note: Print the variable "remaining_50_list" and see if the movie has been added!
-print()
+# # TODO: Hmm. Looks like "remaining_50_list" contains the movie "The Matrix" which is also in the "top_50_list". Let's remove this duplicate title from "remaining_50_list".
+# # Note: Make sure to double check the list you are removing from!
 
+# removed the the matrix from the movies list, correcting the dupe
+remaining_50_list.remove('The Matrix')
+print(remaining_50_list)
 
-# TODO: Let's make sure "remaining_50_list" has exactly 50 movie titles
-print()
+# print()
 
-# TODO: Great! Let's join the two lists into one list called "top_100".
-# Note: Print your new variable to see the combined list of movies
-print()
 
-# TODO: Let's print the first movie in our "top_100" list. Let's use the index lookup method to do this.
-print()
 
-# TODO: Let's print the top 10 movies in "top_100" as a list. Let's use the slice syntax to accomplish this!
-print()
 
-# TODO: Let's convert the list "top_100" back into a string. Assign it to a variable called "top_100_str".
-# Note: Print your new variable to see the string version of the movies
-print()
+# # TODO: We're down to 49 movies in "remaining_50_list". Let's add a movie to the list called "Anywhere But Here"
+# # Note: Print the variable "remaining_50_list" and see if the movie has been added!
+
+# appended the list and added the desired movie
+remaining_50_list.append('Anywhere But Here')
+print(remaining_50_list)
+
+# print()
+
+
+
+# # TODO: Let's make sure "remaining_50_list" has exactly 50 movie titles
+# used len function to get the desired result
+print(len(remaining_50_list))
+
+# print()
+
+
+# # TODO: Great! Let's join the two lists into one list called "top_100".
+# # Note: Print your new variable to see the combined list of movies
+
+# added two previous lists creating a complete list.
+top_100 = remaining_50_list + top_50_list
+print(top_100)
+
+# print()
+
+
+# # TODO: Let's print the first movie in our "top_100" list. Let's use the index lookup method to do this.
+
+# used the index to get desired result
+print(top_100[0])
+
+# print()
+
+# # TODO: Let's print the top 10 movies in "top_100" as a list. Let's use the slice syntax to accomplish this!
+# sliced the top_100 to get desired result
+print(top_100[0:11])
```

Comment:
When you printed [0:11]. how many printed?

[0:10] = index 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 = Ten movies